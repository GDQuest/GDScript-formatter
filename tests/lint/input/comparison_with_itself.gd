#region Good

var x = 5
var y = 10


func good():
	if x == y:
		pass

#endregion

#region Bad

func bad():
	if x == x:
		pass

#endregion
