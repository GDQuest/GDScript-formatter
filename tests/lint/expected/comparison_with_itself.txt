tests/lint/input/comparison_with_itself.gd:16:comparison-with-itself:warning: Redundant comparison 'x == x' - comparing expression with itself